File: JAVA/Daily Flash Codes/Week2/Day2_Jan21/code/prog5.java
/*
 * Program 5 : Write a Program to print following Pattern.
Output :
0
0 0 
0 0 0
0 0 0 0
*/

import java.io.*;

class Program {
	public static void main(String[] args) throws IOException {
		BufferedReader in = new BufferedReader(new InputStreamReader(System.in));		//obj of Bufferedreader to take input
		System.out.println("Enter row: ");
		int row = Integer.parseInt(in.readLine());		//take num of rows
		
		for(int olc=0; olc<row; olc++ )	{
			for(int ilc=0; ilc<=olc; ilc++ )
				System.out.print(" 0 ");
			System.out.println();
		}		
			
	}


}

File: PYTHON/Daily Flash/Week2/Day1_Jan21/code/prog5.py
'''
Program 5 : Write a Program to print following Pattern.
Output :
0
0 0 
0 0 0 
0 0 0 0
'''

row = int(input("Enter row num: "))
for x in range(row):
    for y in range(x+1):
         print("0", end= "  ")
    print()

